Simplenote/SimplenoteConstants.swift

```diff
@@ -31,4 +31,11 @@ class SimplenoteConstants: NSObject {
     static let simplenoteVerificationURL    = currentEngineBaseURL.appendingPathComponent("/account/verify-email/")
     static let simplenoteRequestSignupURL   = currentEngineBaseURL.appendingPathComponent("/account/request-signup")
     static let accountDeletionURL           = currentEngineBaseURL.appendingPathComponent("/account/request-delete/")
+
+    /// Font Constants
+    ///
+    static let NoteFontSizeMinimum = 10
+    static let NoteFontSizeNormal = 15
+    static let NoteFontSizeMaximum = 30
+    static let NoteFontSizeStep = 5
```

Comment:
Since all of these are prefixed as `NoteFontSize`, it would probably be a good idea to move it to its own enum / struct.

How about... `FontSettings`?  (Oh! another note, constants should not be capitalized in Swift) 😄

Reply:
>another note, constants should not be capitalized in Swift)

A shoot that is a copy/paste error on my part.  Will move this to an enum somewhere